s3/src/main/java/com/instaclustr/kafka/connect/s3/AwsStorageConnectorCommonConfig.java

```diff
@@ -69,12 +63,12 @@ public static void verifyS3CredentialsAndBucketInfo(final Map<String, String> se
             s3Client.shutdown();
         } catch (AmazonS3Exception e) {
             switch (e.getErrorCode()) {
-                case "InvalidAccessKeyId":
-                    addErrorMessageToConfigObject(configObject, AWS_ACCESS_KEY_ID, "The defined aws.accessKeyId is invalid");
-                    break;
-                case "SignatureDoesNotMatch":
-                    addErrorMessageToConfigObject(configObject, AWS_SECRET_KEY, "The defined aws.secretKey is invalid");
-                    break;
+	            case "InvalidAccessKeyId":
+	                addErrorMessageToConfigObject(configObject, "", "The AWS AccessKeyId is invalid");
+	                break;
+	            case "SignatureDoesNotMatch":
+	                addErrorMessageToConfigObject(configObject, "", "The AWS SecretKey is invalid");
```

Comment:
These error messages no longer make sense now that aws access key/secret are removed

Reply:
The default chain still picks up the properties:
> Java System Properties - aws.accessKeyId and aws.secretKey

So I think we can even just keep it attached to the AWS_ACCESS_KEY_ID / AWS_SECRET_KEY.

This makes it double silly they went out of their way not to use the default chain...